eclipse/src/saros/ui/views/SarosView.java

```diff
@@ -592,26 +614,29 @@ public static void showNotification(
           @Override
           public void run() {
 
-            if (control != null) {
-              BalloonNotification.showNotification(control, title, text);
+            Control attachToControl = control;
```

Comment:
In my opinion, just using `control` directly here and creating the variable (previously name `sarosViewControl`) below is easier to read. It avoids re-assigning the value for the variable and should not make any functional difference as `control` is a final parameter. (Also, it would allow the call in l.620 to fit into a single line, but that is not really a valid argument.)

But I don't feel very strongly on this, so you can keep it this way if you prefer.

Reply:
The control is declared as final (I do not know if this still a must when using lambdas) so I cannot reassign any value to the control variable.

Reply:
Yeah, what I meant was to just use the `control` variable here and declare the variable `attachToControl` (or whatever you want to call it) later (e.g. in line 631).